packages/auth/README.md

```diff
@@ -0,0 +1,11 @@
+# `@booking-services/auth`
```

Comment:
It will be nice to have some real description here.

Reply:
okay, let me modify it

Reply:
on second thought .... i think the auth package should not exist. because everything that has to do with auth0 has to go through the api-gateway.

I don't know what you think...